src/types.ts

```diff
@@ -25,3 +25,25 @@ export enum SelectionType {
   Hide,
   Highlight,
 }
+
+export enum AgentLevel {
```

Comment:
this is more `checkBoxLevel` than agent level. Sorry if this was unclear, but agents aren't ever children. Checkboxes have children, agents have states